components/Snackbar/src/MDCSnackbarMessageView.m

```diff
@@ -170,9 +165,7 @@ @implementation MDCSnackbarMessageViewButton
 - (instancetype)initWithFrame:(CGRect)frame {
   self = [super initWithFrame:frame];
   if (self) {
-    self.inkMaxRippleRadius = kButtonInkRadius;
     self.inkColor = [UIColor colorWithWhite:1 alpha:(CGFloat)0.06];
-    self.inkStyle = MDCInkStyleUnbounded;
```

Comment:
I'm surprised we're able to delete these values. Were they not necessary with Ink enabled?

Reply:
Yes, the Ink never was supposed to be unbounded, and because unbounded for Ink is broken, it always showed as bounded.

Reply:
Ahh; please move this functionality change out to a separate PR then in case we need to roll it back for any reason.